src/dwarf_parser.cpp

```diff
@@ -415,6 +423,108 @@ ssize_t Dwarf::get_bitfield_size(Dwarf_Die &field_die)
   return 0;
 }
 
+std::optional<std::string> Dwarf::resolve_cu_path(const char *cu_name,
+                                                  const char *cu_comp_dir)
+{
+  if (!cu_name || *cu_name == '\0')
```

Comment:
Make `cu_name` and `cu_comp_dir` be `std::string_view` and you can then use `cu_name.empty()` here.